cms/core/tests/test_views.py

```diff
@@ -18,3 +20,19 @@ def test_csrf_token_mismatch_logs_an_error(self):
             self.client.post("/admin/login/", {})
 
         self.assertIn("CSRF Failure: CSRF cookie", logs.output[0])
+
+
+class TestCookiesBannerTestCase(TestCase):
+    """Test for the cookie banner functionality."""
+
+    def setUp(self):
+        self.client = Client()
+
```

Comment:
```suggestion
```

This is not needed. Django's `TestCase` provides a client which is pretty much the duplicate of this.

The reason the test case above defines it separately is that it needs to enforces CSRF checks, which effectively makes it a custom client